lighthouse-plugin-publisher-ads/audits/cumulative-ad-shift.js

```diff
@@ -153,15 +153,56 @@ class CumulativeAdShift extends Audit {
     };
   }
 
+  /**
+   * Returns a list of layout shift events within the greatest session window.
```

Comment:
Nit: maybe call the method getLayoutShiftEventsInGreatestSessionWindow() or something like that, to make it clear that it's not going to return all of them?

Reply:
Done!